### PR TITLE
[relay-config] is no longer a thing

### PR DESCRIPTION
**Pull request by maraisr**
As of relay 13, the `relay-config` no longer makes sense/has a relevancy. We can safely remove this.

### CODE REVIEW

**Review by levibuzolic — APPROVED** [review]

File: notNeededPackages.json [hunk 1]
```diff
@@ -4475,6 +4475,10 @@
             "libraryName": "rehype-react",
             "asOfVersion": "6.0.0"
         },
+        "relay-config": {
+            "libraryName": "relay-compiler",
+            "asOfVersion": "13.0.0"
```
Comment (jakebailey) [inline notNeededPackages.json:4480]: Hm, this version doesn't actually exist; I'm not sure what affect this'll have. Maybe the package should just be deleted? I'm not sure what precedent there is for effectively deprecated packages, though `relay-compiler` doesn't list itself as deprecated on npm.

(Maybe @sandersn or @andrewbranch know more.)

Comment (maraisr) [reply notNeededPackages.json:4480]: the types for relay-config is no longer a thing. You're right. I just tried to follow the guides, if simply deleting the types, I can do this 🚀

Comment (jakebailey) [reply notNeededPackages.json:4480]: To be clear, the package still exists as v12 on npm, it's just that the package is deprecated, right? I'd think you'd want to deprecate it upstream first.

Comment (maraisr) [reply notNeededPackages.json:4480]: the Facebook maintainers just deleted it from their GitHub repo. I'll reach out and ask if they can deprecate 😅

Comment (jakebailey) [reply notNeededPackages.json:4480]: Oh, well, that certainly doesn't help. I guess it can just be deleted altogether, then.

Comment (maraisr) [reply notNeededPackages.json]: I've deleted the changes to the config file.

**Comment by jakebailey** [comment]
I'm probably showing my confusion, but it sounds like based on the README that the notNeeded config is needed to get the package deprecated. I'll check with the team, though.

**Comment by jakebailey** [comment]
Alright, let's just do this.

**Review by jakebailey — APPROVED** [review]

**Comment by levibuzolic** [comment]
Ready to merge

**Review by levibuzolic — APPROVED** [review]

**Comment by maraisr** [comment]
ready to merge

**Pull request merged**